tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -8,6 +8,9 @@ apps:
   account-control:
     command: bin/run
     plugs: [ account-control ]
+  acrn:
```

Comment:
Apologies if this is a silly question but shouldn't that be `acrn-support` since you changed the name of the interface?